Flow.Launcher.Core/Resource/Theme.cs

```diff
@@ -189,22 +189,11 @@ public ResourceDictionary GetResourceDictionary()
                     new[] { resultItemStyle, resultSubItemStyle, resultItemSelectedStyle, resultSubItemSelectedStyle, resultHotkeyItemStyle, resultHotkeyItemSelectedStyle }, o 
                     => Array.ForEach(setters, p => o.Setters.Add(p)));
             }
-
+            /* Ignore Theme Window Width and use setting */
             var windowStyle = dict["WindowStyle"] as Style;
-
-            var width = windowStyle?.Setters.OfType<Setter>().Where(x => x.Property.Name == "Width")
-                .Select(x => x.Value).FirstOrDefault();
-
-            if (width == null)
-            {
-                windowStyle = dict["BaseWindowStyle"] as Style;
-
-                width = windowStyle?.Setters.OfType<Setter>().Where(x => x.Property.Name == "Width")
-                .Select(x => x.Value).FirstOrDefault();
-            }
-
+            var width = Settings.WindowSize;
+            windowStyle.Setters.Add(new Setter(Window.WidthProperty, width));
```

Comment:
we don't need that right?

Reply:
maybe?

Reply:
Have you remove all the width setting in the theme?

Reply:
I didn't do it yet. Should I?

Reply:
Isn't this pr does that job? https://github.com/Flow-Launcher/Flow.Launcher/pull/755

Reply:
I did so only for the elements inside the window. The themes currently pushed are fixed in window size.

Reply:
ok nvm I remember the setter will be override by direct setting the width. Therefore, it shouldn't be a problem.